test/lib/common.bash

```diff
@@ -23,7 +28,7 @@ __buildPackage() {
 
 	if [[ -n ${BUILDDIR} ]]; then
 		cache=${BUILDDIR}/$(__getCheckSum PKGBUILD)
-		if [[ -d ${cache} ]]; then
+		if __isGlobfile "${cache}"/*"${PKGEXT}"; then
 			cp -Lv ${cache}/*${PKGEXT}{,.sig} ${pkgdest}
```

Comment:
I'm wondering if it makes more sense to use `if cp ... 2>/dev/null; then return 0; fi` here,